### ISSUE TITLE
"docker-entrypoint.sh": executable file not found in $PATH: unknown.

### ISSUE THREAD

**Issue by TheQue42** [goofball222/unifi#109]
## Reporting bugs/issues

Trying to update to latest version (from last latest), but container fails starting. Portainer failed updating, fails with recommended docker-compose up, and a normal docker run results in the same failure.
Running on opensuse 15.4, 
```
LSB Version:    n/a
Distributor ID: openSUSE
Description:    openSUSE Leap 15.4
Release:        15.4
Codename:       n/a

#> docker version
Client:
 Version:           20.10.17-ce
 API version:       1.41
 Go version:        go1.17.13
 Git commit:        a89b84221c85
 Built:             Wed Jun 29 12:00:00 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.17-ce
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.13
  Git commit:       a89b84221c85
  Built:            Wed Jun 29 12:00:00 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.6.6
  GitCommit:        10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-ga916309fff0f
 docker-init:
  Version:          0.1.5_catatonit
  GitCommit:
10:34-Tue_06

```
Tried latest/latest-alpine, as well as latest-ubuntu, with no avail.

**Comment by TheQue42** [comment]
Output from docker-compose --version up -d 
```
[git:master]root@vimes:unifi
#> docker-compose --verbose  up -d
compose.config.config.find: Using configuration files: ./docker-compose.yml
docker.utils.config.find_config_file: Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
docker.utils.config.find_config_file: No config file found
docker.utils.config.find_config_file: Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
docker.utils.config.find_config_file: No config file found
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/version HTTP/1.1" 200 848
compose.cli.command.get_client: docker-compose version 1.25.1, build a82fef0
docker-py version: 4.2.0
CPython version: 3.6.15
OpenSSL version: OpenSSL 1.1.1l  24 Aug 2021 SUSE release 150400.7.16.1
compose.cli.command.get_client: Docker base_url: http+docker://localhost
compose.cli.command.get_client: Docker version: Platform={'Name': ''}, Components=[{'Name': 'Engine', 'Version': '20.10.17-ce', 'Details': {'ApiVersion': '1.41', 'Arch': 'amd64', 'BuildTime': '2022-06-29T12:00:00.000000000+00:00', 'Experimental': 'false', 'GitCommit': 'a89b84221c85', 'GoVersion': 'go1.17.13', 'KernelVersion': '5.14.21-150400.24.33-default', 'MinAPIVersion': '1.12', 'Os': 'linux'}}, {'Name': 'containerd', 'Version': 'v1.6.6', 'Details': {'GitCommit': '10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1'}}, {'Name': 'runc', 'Version': '1.1.4', 'Details': {'GitCommit': 'v1.1.4-0-ga916309fff0f'}}, {'Name': 'docker-init', 'Version': '0.1.5_catatonit', 'Details': {'GitCommit': ''}}], Version=20.10.17-ce, ApiVersion=1.41, MinAPIVersion=1.12, GitCommit=a89b84221c85, GoVersion=go1.17.13, Os=linux, Arch=amd64, KernelVersion=5.14.21-150400.24.33-default, BuildTime=2022-06-29T12:00:00.000000000+00:00
compose.cli.verbose_proxy.proxy_callable: docker inspect_network <- ('unifi_default')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/networks/unifi_default HTTP/1.1" 200 799
compose.cli.verbose_proxy.proxy_callable: docker inspect_network -> {'Attachable': True,
 'ConfigFrom': {'Network': ''},
 'ConfigOnly': False,
 'Containers': {'b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca': {'EndpointID': 'd4a68743cb2fdf498d549132131ab9f00cd5ff82318d1da17f718e7092267bd1',
                                                                                     'IPv4Address': '172.18.0.3/16',
                                                                                     'IPv6Address': '',
                                                                                     'MacAddress': '02:42:ac:12:00:03',
                                                                                     'Name': 'mongodb'}},
 'Created': '2022-02-13T12:21:20.458459348+01:00',
 'Driver': 'bridge',
...
compose.cli.verbose_proxy.proxy_callable: docker info <- ()
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/info HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker info -> {'Architecture': 'x86_64',
 'BridgeNfIp6tables': False,
 'BridgeNfIptables': False,
 'CPUSet': True,
 'CPUShares': True,
 'CgroupDriver': 'cgroupfs',
 'CgroupVersion': '1',
 'ContainerdCommit': {'Expected': '10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1',
                      'ID': '10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1'},
 'Containers': 13,
...
compose.cli.verbose_proxy.proxy_callable: docker inspect_network <- ('unifi_default')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/networks/unifi_default HTTP/1.1" 200 799
compose.cli.verbose_proxy.proxy_callable: docker inspect_network -> {'Attachable': True,
 'ConfigFrom': {'Network': ''},
 'ConfigOnly': False,
 'Containers': {'b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca': {'EndpointID': 'd4a68743cb2fdf498d549132131ab9f00cd5ff82318d1da17f718e7092267bd1',
                                                                                     'IPv4Address': '172.18.0.3/16',
                                                                                     'IPv6Address': '',
                                                                                     'MacAddress': '02:42:ac:12:00:03',
                                                                                     'Name': 'mongodb'}},
 'Created': '2022-02-13T12:21:20.458459348+01:00',
 'Driver': 'bridge',
...
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=False, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=0&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 1605
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 1 items)
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 1605
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 1 items)
compose.cli.verbose_proxy.proxy_callable: docker inspect_container <- ('b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_container -> {'AppArmorProfile': 'docker-default',
 'Args': ['mongod'],
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['mongod'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
                    'GOSU_VERSION=1.12',
...
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=mongo', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dmongo%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 1605
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 1 items)
compose.cli.verbose_proxy.proxy_callable: docker inspect_container <- ('b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_container -> {'AppArmorProfile': 'docker-default',
 'Args': ['mongod'],
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['mongod'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
                    'GOSU_VERSION=1.12',
...
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
compose.cli.verbose_proxy.proxy_callable: docker inspect_image <- ('mongo')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/images/mongo/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_image -> {'Architecture': 'amd64',
 'Author': '',
 'Comment': '',
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['mongod'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
...
compose.cli.verbose_proxy.proxy_callable: docker inspect_image <- ('goofball222/unifi:latest-ubuntu')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/images/goofball222/unifi:latest-ubuntu/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_image -> {'Architecture': 'amd64',
 'Author': '',
 'Comment': 'buildkit.dockerfile.v0',
 'Config': {'ArgsEscaped': True,
            'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['unifi'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
...
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=mongo', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dmongo%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 1605
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 1 items)
compose.cli.verbose_proxy.proxy_callable: docker inspect_image <- ('mongo')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/images/mongo/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_image -> {'Architecture': 'amd64',
 'Author': '',
 'Comment': '',
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['mongod'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
...
compose.cli.verbose_proxy.proxy_callable: docker inspect_container <- ('b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/b718a09d5aecb1c5b9fae6b6e5e05d125a9766a3ce7efd5a7432a2787a1af7ca/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_container -> {'AppArmorProfile': 'docker-default',
 'Args': ['mongod'],
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['mongod'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
                    'GOSU_VERSION=1.12',
...
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
compose.parallel.feed_queue: Pending: {<Service: unifi>, <Service: mongo>}
compose.parallel.feed_queue: Starting producer thread for <Service: unifi>
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
compose.parallel.feed_queue: Starting producer thread for <Service: mongo>
compose.service.execute_convergence_plan: mongodb is up-to-date
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
compose.cli.verbose_proxy.proxy_callable: docker containers <- (all=True, filters={'label': ['com.docker.compose.project=unifi', 'com.docker.compose.service=unifi', 'com.docker.compose.oneoff=False']})
compose.parallel.parallel_execute_iter: Finished processing: <Service: mongo>
compose.parallel.feed_queue: Pending: set()
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/json?limit=-1&all=1&size=0&trunc_cmd=0&filters=%7B%22label%22%3A+%5B%22com.docker.compose.project%3Dunifi%22%2C+%22com.docker.compose.service%3Dunifi%22%2C+%22com.docker.compose.oneoff%3DFalse%22%5D%7D HTTP/1.1" 200 3
compose.cli.verbose_proxy.proxy_callable: docker containers -> (list with 0 items)
Creating unifi-network ...
compose.parallel.feed_queue: Pending: {ServiceName(project='unifi', service='unifi', number=1)}
compose.parallel.feed_queue: Starting producer thread for ServiceName(project='unifi', service='unifi', number=1)
compose.cli.verbose_proxy.proxy_callable: docker inspect_image <- ('goofball222/unifi:latest-ubuntu')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/images/goofball222/unifi:latest-ubuntu/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_image -> {'Architecture': 'amd64',
 'Author': '',
 'Comment': 'buildkit.dockerfile.v0',
 'Config': {'ArgsEscaped': True,
            'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['unifi'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
...
compose.cli.verbose_proxy.proxy_callable: docker inspect_image <- ('goofball222/unifi:latest-ubuntu')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/images/goofball222/unifi:latest-ubuntu/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_image -> {'Architecture': 'amd64',
 'Author': '',
 'Comment': 'buildkit.dockerfile.v0',
 'Config': {'ArgsEscaped': True,
            'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['unifi'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
...
compose.service.build_container_labels: Added config hash: 7954f6b8f9c2fba4f719cb6f090b4811148aba05b687e6c6d7e92a075481ecdc
compose.cli.verbose_proxy.proxy_callable: docker create_host_config <- (links=[], port_bindings={'3478/udp': ['3478'], '6789/tcp': ['6789'], '8080/tcp': ['8080'], '8443/tcp': ['8443'], '8880/tcp': ['8880'], '8843/tcp': ['8843'], '1900/udp': [('10.9.24.20', '1900')], '10001/udp': [('10.9.24.20', '10001')]}, binds=['/pods/unifi-network/logs:/usr/lib/unifi/logs:rw', '/pods/unifi-network/data:/usr/lib/unifi/data:rw', '/pods/unifi-network/cert:/usr/lib/unifi/cert:rw', '/etc/localtime:/etc/localtime:ro'], volumes_from=[], privileged=False, network_mode='unifi_default', devices=None, dns=None, dns_opt=None, dns_search=None, restart_policy={'Name': 'always', 'MaximumRetryCount': 0}, runtime=None, cap_add=['NET_ADMIN', 'SYS_ADMIN', 'NET_RAW'], cap_drop=None, mem_limit=None, mem_reservation=None, memswap_limit=None, ulimits=None, log_config={'Type': '', 'Config': {}}, extra_hosts=None, read_only=None, pid_mode=None, security_opt=None, ipc_mode=None, cgroup_parent=None, cpu_quota=None, shm_size=None, sysctls=None, pids_limit=None, tmpfs=[], oom_kill_disable=None, oom_score_adj=None, mem_swappiness=None, group_add=None, userns_mode=None, init=None, init_path=None, isolation=None, cpu_count=None, cpu_percent=None, nano_cpus=None, volume_driver=None, cpuset_cpus=None, cpu_shares=None, storage_opt=None, blkio_weight=None, blkio_weight_device=None, device_read_bps=None, device_read_iops=None, device_write_bps=None, device_write_iops=None, mounts=None, device_cgroup_rules=None, cpu_period=None, cpu_rt_period=None, cpu_rt_runtime=None)
compose.cli.verbose_proxy.proxy_callable: docker create_host_config -> {'Binds': ['/pods/unifi-network/logs:/usr/lib/unifi/logs:rw',
           '/pods/unifi-network/data:/usr/lib/unifi/data:rw',
           '/pods/unifi-network/cert:/usr/lib/unifi/cert:rw',
           '/etc/localtime:/etc/localtime:ro'],
 'CapAdd': ['NET_ADMIN', 'SYS_ADMIN', 'NET_RAW'],
 'Links': [],
 'LogConfig': {'Config': {}, 'Type': ''},
 'NetworkMode': 'unifi_default',
 'PortBindings': {'10001/udp': [{'HostIp': '10.9.24.20', 'HostPort': '10001'}],
                  '1900/udp': [{'HostIp': '10.9.24.20', 'HostPort': '1900'}],
...
compose.cli.verbose_proxy.proxy_callable: docker create_container <- (environment=['DB_MONGO_LOCAL=false', 'DB_MONGO_URI=mongodb://mongodb:27017/unifi', 'STATDB_MONGO_URI=mongodb://mongodb:27017/unifi_stat', 'TZ=Europe/Stockholm', 'UNIFI_DB_NAME=unifi', 'DEBUG=true'], image='goofball222/unifi:latest-ubuntu', labels={'com.centurylinklabs.watchtower.enable': 'false', 'com.docker.compose.project': 'unifi', 'com.docker.compose.service': 'unifi', 'com.docker.compose.oneoff': 'False', 'com.docker.compose.project.working_dir': '/root/bin/docker/unifi', 'com.docker.compose.project.config_files': 'docker-compose.yml', 'com.docker.compose.container-number': '1', 'com.docker.compose.version': '1.25.1', 'com.docker.compose.config-hash': '7954f6b8f9c2fba4f719cb6f090b4811148aba05b687e6c6d7e92a075481ecdc'}, ports=[('3478', 'udp'), ('6789', 'tcp'), ('8080', 'tcp'), ('8443', 'tcp'), ('8880', 'tcp'), ('8843', 'tcp'), ('1900', 'udp'), ('10001', 'udp')], volumes={'/usr/lib/unifi/logs': {}, '/usr/lib/unifi/data': {}, '/usr/lib/unifi/cert': {}, '/etc/localtime': {}}, name='unifi-network', detach=True, host_config={'NetworkMode': 'unifi_default', 'RestartPolicy': {'Name': 'always', 'MaximumRetryCount': 0}, 'CapAdd': ['NET_ADMIN', 'SYS_ADMIN', 'NET_RAW'], 'VolumesFrom': [], 'Binds': ['/pods/unifi-network/logs:/usr/lib/unifi/logs:rw', '/pods/unifi-network/data:/usr/lib/unifi/data:rw', '/pods/unifi-network/cert:/usr/lib/unifi/cert:rw', '/etc/localtime:/etc/localtime:ro'], 'PortBindings': {'3478/udp': [{'HostIp': '', 'HostPort': '3478'}], '6789/tcp': [{'HostIp': '', 'HostPort': '6789'}], '8080/tcp': [{'HostIp': '', 'HostPort': '8080'}], '8443/tcp': [{'HostIp': '', 'HostPort': '8443'}], '8880/tcp': [{'HostIp': '', 'HostPort': '8880'}], '8843/tcp': [{'HostIp': '', 'HostPort': '8843'}], '1900/udp': [{'HostIp': '10.9.24.20', 'HostPort': '1900'}], '10001/udp': [{'HostIp': '10.9.24.20', 'HostPort': '10001'}]}, 'Links': [], 'LogConfig': {'Type': '', 'Config': {}}}, networking_config={'EndpointsConfig': {'unifi_default': {'Aliases': ['unifi'], 'IPAMConfig': {}}}})
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
urllib3.connectionpool._make_request: http://localhost:None "POST /v1.25/containers/create?name=unifi-network HTTP/1.1" 201 88
compose.cli.verbose_proxy.proxy_callable: docker create_container -> {'Id': 'c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6',
 'Warnings': []}
compose.cli.verbose_proxy.proxy_callable: docker inspect_container <- ('c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6')
urllib3.connectionpool._make_request: http://localhost:None "GET /v1.25/containers/c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6/json HTTP/1.1" 200 None
compose.cli.verbose_proxy.proxy_callable: docker inspect_container -> {'AppArmorProfile': '',
 'Args': ['unifi'],
 'Config': {'AttachStderr': False,
            'AttachStdin': False,
            'AttachStdout': False,
            'Cmd': ['unifi'],
            'Domainname': '',
            'Entrypoint': ['docker-entrypoint.sh'],
            'Env': ['DB_MONGO_LOCAL=false',
                    'DB_MONGO_URI=mongodb://mongodb:27017/unifi',
...
compose.cli.verbose_proxy.proxy_callable: docker disconnect_container_from_network <- ('c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6', 'unifi_default')
urllib3.connectionpool._make_request: http://localhost:None "POST /v1.25/networks/unifi_default/disconnect HTTP/1.1" 200 0
compose.cli.verbose_proxy.proxy_callable: docker disconnect_container_from_network -> None
compose.cli.verbose_proxy.proxy_callable: docker connect_container_to_network <- ('c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6', 'unifi_default', aliases=['c18d25642692', 'unifi'], ipv4_address=None, ipv6_address=None, links=[], link_local_ips=None)
urllib3.connectionpool._make_request: http://localhost:None "POST /v1.25/networks/unifi_default/connect HTTP/1.1" 200 0
compose.cli.verbose_proxy.proxy_callable: docker connect_container_to_network -> None
compose.cli.verbose_proxy.proxy_callable: docker start <- ('c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6')
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
compose.parallel.feed_queue: Pending: set()
urllib3.connectionpool._make_request: http://localhost:None "POST /v1.25/containers/c18d25642692cedb1769d2d01ff3efe4a0d370f1744fffa6d7b52a6550e34cb6/start HTTP/1.1" 400 200
Creating unifi-network ... error
compose.parallel.feed_queue: Pending: set()

ERROR: for unifi-network  Cannot start service unifi: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "docker-entrypoint.sh": executable file not found in $PATH: unknown
compose.parallel.parallel_execute_iter: Failed: <Service: unifi>
compose.parallel.feed_queue: Pending: set()

ERROR: for unifi  Cannot start service unifi: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "docker-entrypoint.sh": executable file not found in $PATH: unknown
ERROR: compose.cli.main.main: Encountered errors while bringing up the project.
10:37-Tue_06
[git:master]root@vimes:unifi
#>
```

**Comment by TheQue42** [comment]
On a side note, ever since since 7.2 (I _think_) I have not been able to run unifi with the default mongo/ image as is listed in the compose example, but instead I have to fix the version to 5.0. Am I doing something wrong? Should it work with mongo/latest?

**Comment by TheQue42** [comment]
> Looks like the script file modes got changed/overwritten somewhere during the creation & merge of PR #108. I just fixed those and pushed the change, latest image is rebuilding now, should be available in 10-15 minutes.
> 
> As far as MongoDB > 5.0, I ran into that myself. The Mongo client built into UniFi is old, and is using interfaces/functions deprecated in versions of MongoDB beyond 5.0. Unfortunately, since it's built into UniFi itself I can't fix it in the container, it's dependent on UI to upgrade at some point. My fix was to pin my mongo containers in the stack to mongo:5.0, similar to what you found.

Thanks a million for a quick fix. Regarding the mongo:5 issue, then maybe update the docker-compose example to use the :5 version? Since it wasnt updated, I started wondering what I was doing wrong :-)
